std/range/package.d

```diff
@@ -2887,13 +2905,37 @@ private:
     UT _value;
 
 public:
-    @property inout(T) front() inout { return _value; }
-    @property inout(T) back() inout { return _value; }
+    ///
+    @property inout(T) front() inout
+    {
+        return _value;
+    }
+
+    ///
```

Comment:
> I see a mix of both in the docs, I'm just not a fan of docstrings with no words

Let's wait for feedback from other people here - I am more on the side of supplying unnecessary information bloats and is less helpful.